.github/workflows/checks.yaml

```diff
@@ -58,6 +58,7 @@ jobs:
         with:
           version: v1.55.2
           args: --timeout 10m0s
+          skip-pkg-cache: true
```

Comment:
it seems unrelated to the PR. Can you revert?

Reply:
Thanks. Fixed it.